war_or_peace_runner.rb

```diff
@@ -0,0 +1,50 @@
+# not required
+require './lib/card'
+require './lib/deck'
+require './lib/turn'
+require './lib/player'
+require 'pry'
+# required
+require './lib/game'
+
+# all_cards = game.create_full_deck
+# all_cards.shuffle!
+
+# player1 = Player.new('Megan', deck1)
+# player2 = Player.new('Aurora', deck2)
+
+turn = Turn.new(player1, player2)
+game = Game.new(turn)
```

Comment:
Your `Game` class doesn't initialize with an argument so we don't need to pass `turn` here. This means we can also get rid of line 16. Your `Game` creates the players so we don't need to do that inside the runner.